source/CommandLine/OptionParsing/Option.cs

```diff
@@ -0,0 +1,311 @@
+//
```

Comment:
These were all inside one `Options.cs` previously. I've split them out to separate files.